style.css

```diff
@@ -0,0 +1,21 @@
+
+
+#title {color: blue;}
+#main{  background-color: gray;
+}
+#image {max-width:100%; height:auto;
```

Comment:
[OPTIONAL] It's very good in requesting this change, however, it will be very nice if you could add emojis to your inline comments. Emojis can enhance communication by adding visual cues to your comments. They can help convey emotions, tone, or intent that may not be immediately apparent from text alone. For example, using a thumbs-up emoji can indicate approval or agreement, while a confused face emoji can indicate that something is unclear. :slightly_smiling_face: